apps/frontend/src/App.css

```diff
@@ -1,7 +1,5 @@
-#root {
-  max-width: 1280px;
-  margin: 40px;
-  padding: 2rem;
-  text-align: center;
-  background-color: "white";
```

Comment:
Does this not impact the rest of the frontend?

Reply:
it might, but this change is correct and front end should not have included styles here before. They might have been leftover from the initial project creation